terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -0,0 +1,1261 @@
+  ssh_key_path            = "./salt/controller/id_rsa.pub"
+}
+
+module "debian11-minion" {
+  source             = "./modules/minion"
+  base_configuration = module.base_core.configuration
+  product_version    = "head"
+  name               = "min-debian11"
+  image              = "debian11o"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:6e"
+    memory             = 4096
+  }
+
+  server_configuration = {
+    hostname = "suma-bv-50-srv.mgr.suse.de"
+  }
+  auto_connect_to_master  = false
+  use_os_released_updates = false
+  ssh_key_path            = "./salt/controller/id_rsa.pub"
+}
+
+module "debian12-minion" {
+  source             = "./modules/minion"
+  base_configuration = module.base_core.configuration
+  product_version    = "head"
+  name               = "min-debian12"
+  image              = "debian12o"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:6c"
+    memory             = 4096
+  }
+
+  server_configuration = {
+    hostname = "suma-bv-50-srv.mgr.suse.de"
+  }
+  auto_connect_to_master  = false
+  use_os_released_updates = false
+  ssh_key_path            = "./salt/controller/id_rsa.pub"
+
+  additional_packages = [ "venv-salt-minion" ]
+  install_salt_bundle = true
+}
+
+// WORKAROUND: overdrive3 will be replaced with a new ARM server
+//module "opensuse154arm-minion" {
+//  providers = {
+//    libvirt = libvirt.overdrive3
+//  }
+//  source             = "./modules/minion"
+//  base_configuration = module.base_arm.configuration
+//  product_version    = "head"
+//  name               = "min-o154arm-n"
+//  image              = "opensuse154armo"
+//  provider_settings = {
+//    mac                = "aa:b2:92:42:00:6f"
+//    memory             = 2048
+//    vcpu               = 2
+//    xslt               = file("../../susemanager-ci/terracumber_config/tf_files/common/tune-aarch64.xslt")
+//  }
+//  server_configuration = {
+//    hostname = "suma-bv-50-srv.mgr.suse.de"
+//  }
+//  auto_connect_to_master  = false
+//  use_os_released_updates = false
+//  ssh_key_path            = "./salt/controller/id_rsa.pub"
+//}
+
+// WORKAROUND: overdrive3 will be replaced with a new ARM server
+//module "opensuse155arm-minion" {
+//  providers = {
+//    libvirt = libvirt.overdrive3
+//  }
+//  source             = "./modules/minion"
+//  base_configuration = module.base_arm.configuration
+//  product_version    = "head"
+//  name               = "min-o155arm-n"
+//  image              = "opensuse155armo"
+//  provider_settings = {
+//    mac                = "aa:b2:92:42:00:70"
+//    memory             = 2048
+//    vcpu               = 2
+//    xslt               = file("../../susemanager-ci/terracumber_config/tf_files/common/tune-aarch64.xslt")
+//  }
+//  server_configuration = {
+//    hostname = "suma-bv-50-srv.mgr.suse.de"
+//  }
+//  auto_connect_to_master  = false
+//  use_os_released_updates = false
+//  ssh_key_path            = "./salt/controller/id_rsa.pub"
+//}
+
```

Comment:
s390 one

Reply:
We do not have DNS/DHCP entries for them, so I left s390 out.